magenta/music/musicxml_parser_test.py

```diff
@@ -300,5 +313,40 @@ def testFluteScale(self):
     self.maxDiff = None
     self.assertProtoEquals(expected_ns, ns)
 
+  def testatonaltransposition(self):
```

Comment:
function names should be lowercase with words separated by underscores: `def test_atonal_transposition`. We've been a little inconsistent with that ourselves, but we should make sure new methods follow the correct pattern.

https://google.github.io/styleguide/pyguide.html?showone=Comments#Naming